lib/config/presets/bitbucket-server/index.ts

```diff
@@ -17,11 +17,16 @@ const http = new BitbucketServerHttp();
 export async function fetchJSONFile(
   repo: string,
   fileName: string,
-  endpoint: string
+  endpoint: string,
+  packageTag?: string
```

Comment:
```suggestion
  branchOrTag?: string
```

Should match interface declaration